src/main/java/kr/syeyoung/dungeonsguide/features/FeatureParameter.java

```diff
@@ -48,4 +70,11 @@ public void setToDefault() {
     public T getValue() {
         return value == null ? default_value : value;
     }
+
+    public void setValue(T newValue){
```

Comment:
this opens room for state inconsistency.... maybe revert back?
We can discuss this later